doc/nrf/known_issues.rst

```diff
@@ -244,6 +244,22 @@ Missing :file:`CMakeLists.txt`
 Thread
 ======
 
+.. rst-class:: v1-6-1 v1-6-0
+
+KRKNWK-10633: Link Metrics Enhanced ACK-based Probing does not work properly.
+ **Workaround:** Not available, issue is fixed in newer version.
+
+.. rst-class:: v1-6-1 v1-6-0
+
+KRKNWK-10467: The current implementation does not guarantee that all retransmitted frames will be secured when using the radio driver transmission security capabilities.
+  For this reason, OpenThread retransmissions are disabled by default when the :option:`CONFIG_NRF_802154_ENCRYPTION` Kconfig option is enabled.
+  You can enable the retransmissions at your own risk.
+
+.. rst-class:: v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0
```

Comment:
```suggestion
.. rst-class:: v1-6-1 v1-6-0
```
Older versions are not affected since the bug was introduced recently by: https://github.com/openthread/openthread/commit/257188b5e94687b63391602b532b08a4a9681a5d

Reply:
fixed.